### PR TITLE
Introduce KubeEtcd3Down and coment out the HA KubeEtcd3InsufficientPeers

### PR DESCRIPTION
**Pull request by mvladev**

### CODE REVIEW

File: charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml [hunk 1]
```diff
@@ -2,19 +2,30 @@ groups:
...
+    expr: absent(up{job="kube-etcd3",role="main"}) or absent(up{job="kube-etcd3",role="events"})
     for: 3m
     labels:
       service: etcd
       severity: critical
       type: seed
     annotations:
-      description: If one more etcd3 peer goes down the etc3 cluster {{ $labels.role }} will be unavailable
-      summary: etcd3 cluster small
+      description: If etc3 cluster {{ $labels.role }} is be unavailable or cannot be scrapped
```
Comment (rfranzke) [inline charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml]: *either
*etcd3
*remove "be"

File: charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml [hunk 2]
```diff
@@ -2,19 +2,30 @@ groups:
...
+      summary: etcd3 cluster down
+
+  # TODO: enable this alert for HA scenarios
```
Comment (rfranzke) [inline charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml]: Not sure why we keep that whole section; there are currently no plans to do anything regarding HA?

File: charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml [hunk 3]
```diff
@@ -2,19 +2,30 @@ groups:
...
   rules:
 
-  # alert if another failed peer will result in an unavailable cluster
-  - alert: KubeEtcd3InsufficientPeers
-    expr: absent(up{job="kube-etcd3",role="main"})
-      or absent(up{job="kube-etcd3",role="events"})
-      or count(up{job="kube-etcd3"} == 0) by(role) > (count(up{job="kube-etcd3"}) by (role) / 2 - 1)
+  # alert if etcd is down
+  - alert: KubeEtcd3Down
+    expr: absent(up{job="kube-etcd3",role="main"}) or absent(up{job="kube-etcd3",role="events"})
```
Comment (rfranzke) [inline charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml:7]: Just for me (not being too familiar with the alert definition language): What is this `job="kube-etcd"` standing for?

**Review by rfranzke — COMMENTED** [review]
Thanks, have added some suggestions.

File: charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml (hunk 3 above)
Comment (mvladev) [reply charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml:7]: In Prometheus terms, an endpoint you can scrape is called an instance, usually corresponding to a single process. A collection of instances with the same purpose, a process replicated for scalability or reliability for example, is called a job.

For example, an API server job with four replicated instances:

    job: api-server
        instance 1: 1.2.3.4:5670
        instance 2: 1.2.3.4:5671
        instance 3: 5.6.7.8:5670
        instance 4: 5.6.7.8:5671

Automatically generated labels and time series

When Prometheus scrapes a target, it attaches some labels automatically to the scraped time series which serve to identify the scraped target:

    job: The configured job name that the target belongs to.
    instance: The <host>:<port> part of the target's URL that was scraped.

`up{job="kube-etcd3"}` means - find a time series metric `up` for which the value of the `job` label is `kube-etcd3`

Comment (rfranzke) [reply charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml:7]: OK, thank you for that explanation. And [here](https://github.com/gardener/gardener/blob/d318acd0ba5099a94b2c1d324af55533c060d228/charts/seed-controlplane/charts/etcd/templates/etcd-statefulset.yaml#L33) the job name is defined on the etcd service. That's how Prometheus knows which pods/services are relevant. Thanks!

**Review by rfranzke — APPROVED** [review]

**Pull request merged**